### ISSUE TITLE
Homepage Illustrations

### ISSUE THREAD

**Issue by hawkticehurst** [CouncilDataProject/cdp-frontend#39]
Per the UX Research [recommendations](https://docs.google.com/presentation/d/15rkic20QV6GU0_nL-8zHIgcw8-o1bCQIiYMsY2j6qFg/edit#slide=id.g8bf3178c59_0_65) having some form of friendly illustrations that can make the homepage more approachable and potentially add some personality would be great.

Whether this is creating illustrations or sourcing high-quality illustrations **with permissive licensing** we are open to help in this department.

@JacksonMaxfield @tohuynh Additional thoughts and feelings?

**Comment by JacksonMaxfield** [comment]
Yea I think both the UX teams illustrations and the illustration currently up on [councildataproject.github.io](https://councildataproject.github.io) are from [undraw](https://undraw.co/)

That said. I am not entirely convinced on illustrations on the landing page. I agree that we should make it more approachable but I think I need to see some landing page experiments first.

**Comment by hawkticehurst** [comment]
Also inserting my opinion I'm going to say if we could avoid using https://www.humaaans.com/ that would be nice. While I absolutely adore that website I think these illustrations are overused at this point and would detract from a sense of unique personality for CDP.

With that said, this isn't a super strong opinion so others are more than welcome to disagree if there's an illustration that feels particularly fitting for CDP.

**Comment by JacksonMaxfield** [comment]
Hah. Yea. I think we have similar ideas.

CDP should largely be devoid of illustrations imo. I think we can be more approachable though accent colors and iconography and such or maybe an illustration _once_ but that's about it. I remember when the feedback came back from the UX interviews, a couple of people said they appreciated that because the design of the current site was _so_ minimal they trusted it more.

I think that is what we should strive for. Minimal to the point of "just the facts" / "pure utility" with accents here and there to make it more approachable and unique in our own style.

But that is also just classic modern tech stuff so maybe I have just gone off the deep end.

**Comment by hawkticehurst** [comment]
I think we do indeed have similar ideas.

Illustrations in my mind might actually be more akin to subtle accent colors and iconography that generally raise the baseline aesthetic or polish of the website versus an overly dramatic hero illustration.

I think the goal should be to convey a sense of credibility and polish without getting in the way of the user.

So with that thinking should I potentially relabel this issue @JacksonMaxfield?

**Comment by JacksonMaxfield** [comment]
Yeah I think can just be relabeled to probably landing page container? The search + filters stuff I mentioned in the other issue?

**Comment by JacksonMaxfield** [comment]
Errr I guess? Search container / Hero component are probably different? Idk how to think about that

**Comment by JacksonMaxfield** [comment]
In which case, this depends on #37

**Comment by tohuynh** [comment]
I'm OK with illustrations if they convey information that's not already on the page. For example, I don't think the search illustration adds much since there's already a looking-glass search icon button and the word 'Search' throughout the landing page.

**Comment by BrianL3** [comment]
Friendly illustrations can serve more than just an aesthetic purpose.  The "purely functional" look can lead to an intimidating "Wall of Text" that really detracts from usability (and certainly may scare away people who are not being task-directed in a usability study!).  Some things really are easier shown than read.
